Libs.sln

```diff
@@ -248,6 +250,34 @@ Global
 		{D02BB914-D0E3-45F5-A68B-59D079FA742E}.Release|x64.Build.0 = Release|Any CPU
 		{D02BB914-D0E3-45F5-A68B-59D079FA742E}.Release|x86.ActiveCfg = Release|Any CPU
 		{D02BB914-D0E3-45F5-A68B-59D079FA742E}.Release|x86.Build.0 = Release|Any CPU
+		{0CE6E6B7-B914-4198-9C32-3B85CA4C2567}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I tried to leave the "ref" project out of the SLN, but I'm getting all sort of errors :(